clarifai/models/model_serving/models/output.py

```diff
@@ -72,8 +72,8 @@ def __post_init__(self):
     """
     Validate input upon initialization.
     """
-    assert self.predicted_text.ndim == 1, \
-      f"All predictions must be 1-dimensional, Got text-dims: {self.predicted_text.ndim} instead."
+    assert self.predicted_text.ndim == 0, \
+      f"All predictions must be 0-dimensional, Got text-dims: {self.predicted_text.ndim} instead."
```

Comment:
This seems inconsistent with the text-to-text definition that says outputs dims are [1] ? Or I may be understanding this assertion wrong

Reply:
It must be 0 otherwise will get error of `EAGLE-3447`